raw_topic: raw
temperature_topic: celsius
image_topic: image
display_min_max: False
use_colormap: True
